scss/components/listing.scss

```diff
@@ -56,6 +56,10 @@ table.listing {
             &.room-for-numbers {
                 padding-right: 80px;
             }
+            &.profile {
+                max-width: 384px;
+            }
```

Comment:
I'm not convinced this is the right solution. Hover doesn't clobber the left lower metadata on the homepage:

![screen shot 2017-03-02 at 12 41 53 pm](https://cloud.githubusercontent.com/assets/134455/23519669/bdef4ea4-ff45-11e6-87e2-48fb46bc0216.png)


but it does here:

![screen shot 2017-03-02 at 12 42 02 pm](https://cloud.githubusercontent.com/assets/134455/23519660/b6fe7d22-ff45-11e6-9ba3-6db7728d892b.png)